components/director/internal/metrics/pusher.go

```diff
@@ -28,7 +30,7 @@ func NewPusher(endpoint string, timeout time.Duration) *Pusher {
 	}, []string{"method", "code", "desc"})
 
 	instanceID := uuid.New()
-	log.WithField(InstanceIDKeyName, instanceID).Infof("Initializing Metrics Pusher...")
+	log.D().WithField(InstanceIDKeyName, instanceID).Infof("Initializing Metrics Pusher...")
```

Comment:
Why do you decided to use default logger here and not to propagate a context to the pusher?

Reply:
Pusher is created only in TenantFetcher. The TenantFetcher uses context.Background() and ads to it log with configuration. Then this configuration is applyed to the default logger as there is no logger in this context. Nothing else is added to the context in TenantFetcher. So passing context to pusher and using the logger from it will be equivalent to using the default logger directly.